src/content/en/2021/structured-data.md

```diff
@@ -1,14 +1,16 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Structured Data
-description: TODO
-authors: []
-reviewers: []
-analysts: []
-editors: []
-translators: []
-results: https://docs.google.com/spreadsheets/d/1uEA217YjpX1xdRVBnIz9ekjpjIrpGIh5GwUxvnzJNig/
-featured_quote: TODO
+description: Structured Data chapter of the 2021 Web Almanac covering adoption and impact of schema.org, RDFa, Microdata and more.
+authors: [jonoalderson, cyberandy]
+reviewers: [vdwijngaert, philbarker]
+analysts: [GregBrimble]
+translators: [tunetheweb]
+editors: [jvandriel, JasmineDWillson]
+results: https://docs.google.com/spreadsheets/d/1uEA217YjpX1xdRVBnIz9ekjpjIrpGIh5GwUxvnzJNig/edit?usp=sharing
+jonoalderson_bio: Jono Alderson is a digital strategist, marketing technologist, and full stack developer. He enjoys dabbling with website performance, technical SEO, schema.org and all things structured data.
+cyberandy_bio: Andrea Volpini is the CEO of WordLift, and is currently focusing on the semantic web, SEO and artificial intelligence.
+featured_quote: Describing our content using structured data enables machines to treat webpages and websites as databases. That creates a wealth of possibilities for business, technology, and society.
```

Comment:
Please fill in featured stats below this.

Reply:
Done!